query_helper.py

```diff
@@ -44,6 +50,38 @@
     "SELECT COUNT(key) FROM {table} "
     "WHERE transaction_et = '{date}';")
 
+_REDSHIFT_CODE_COUNT_QUERY = \
+    'SELECT COUNT({code}) FROM {table} WHERE deletion_date IS NULL;'
+
+_REDSHIFT_DISTINCT_CODE_COUNT_QUERY = \
+    'SELECT COUNT(DISTINCT {code}) FROM {table} WHERE deletion_date IS NULL;'
```

Comment:
Almost certainly an over-optimization, but these could be combined into a single query. If they're expensive it might help?

Reply:
Good point -- done!